install_config/registry/accessing_registry.adoc

```diff
@@ -305,4 +306,42 @@ openshift_registry_request_duration_seconds_count{name="test/origin-pod",operati
 <1> `<user>` can be arbitrary, but `<secret>` must match the value specified in the
 xref:../registry/extended_registry_configuration.adoc#docker-registry-configuration-reference-openshift[registry configuration].
 
+Another how to access the metrics is to use a cluster role. You still need to enable the endpoint,
+but you do not need to specify a `<secret>`. The part of configuration file responsible
+for the metrics should look like this:
```

Comment:
"The part of the configuration file responsible for metrics should look like this:"